### PR TITLE
Syntax changes

### PR DESCRIPTION
**Pull request by cquiroz**
Add a method to use pprint as Callback

### CODE REVIEW

**Review by toddburnside — APPROVED** [review]
LGTM

File: common/src/main/scala/explore/syntax/ui/package.scala [hunk 1]
```diff
@@ -1,80 +1,55 @@
...
-      val newClazz = input.clazz |+| ExploreStyles.ClearableInputPaddingReset
-      input.copy(icon = clearInputIcon[EV, A](input.value), clazz = newClazz)
-    }
-  }
-
-  extension [A](c: js.UndefOr[A => Callback])
-    def toJs: js.UndefOr[js.Function1[A, Unit]] = c.map(x => (a: A) => x(a).runNow())
-
-}
+extension (c: Callback.type) def pprintln[T](a: T): Callback = Callback(_root_.pprint.pprintln(a))
```
Comment (cquiroz) [inline common/src/main/scala/explore/syntax/ui/package.scala:55]: I think I'll move this to lucuma-ui

Comment (rpiaggio) [reply common/src/main/scala/explore/syntax/ui/package.scala:55]: Nice.

Can we have a `Logger.pdebug` too, please?

Comment (cquiroz) [reply common/src/main/scala/explore/syntax/ui/package.scala:55]: We have this too, is it in use?
https://github.com/gemini-hlsw/explore/blob/8f7d99556e53adc49f073e53184b258a7d26667a/common/src/main/scala/explore/utils/pprinter.scala

Comment (rpiaggio) [reply common/src/main/scala/explore/syntax/ui/package.scala:55]: I had completely forgotten about that.  It doesn't seem to be in use.

**Pull request merged**